docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

```diff
@@ -2,23 +2,21 @@
 
 = Stopping infrastructure management and Compute services
 
-The source cloud's control plane can be decomissioned,
-which is taking down only cloud controllers, database and messaging nodes.
-Nodes that must remain functional are those running the `Compute`, `Storage`,
-or `Networker` roles (in terms of composable roles covered by
-{OpenStackPreviousInstaller} Heat Templates).
+You must stop cloud Controller nodes, database nodes, and messaging nodes on the {rhos_prev_long} {rhos_prev_ver} control plane. However, nodes that are running the Compute, Storage, or Networker roles must remain functional on the control plane.
```

Comment:
Maybe instead of "must remain functional", could say something more explicit like:
"Do not stop nodes that are running the Compute, Storage, or Networker roles on the control plane."